packages/terra-application-layout/src/terra-dev-site/doc/common/ExtensionsDisclosure.jsx

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 import { withDisclosureManager, disclosureManagerShape } from 'terra-disclosure-manager';
 import ContentContainer from 'terra-content-container';
 import ActionHeader from 'terra-action-header';
+import classNames from 'classnames/bind';
+import styles from './ExtensionsDisclosure.scss';
```

Comment:
Lets update this to use `./ApplicationLayoutDocCommon';` directly

Reply:
Done